t/plugin/google-cloud-logging.t

```diff
@@ -745,9 +783,58 @@ passed
 
 
 
-=== TEST 25: test route(https file configuration SSL authentication succeed: ssl_verify = false)
+=== TEST 25: test route (https file configuration SSL authentication succeed: ssl_verify = false)
 --- request
 GET /hello
 --- wait: 2
 --- response_body
 hello world
+
+
+
+=== TEST 26: set fetch request body true in configuration
+--- config
+    location /t {
+        content_by_lua_block {
+            local config = {
+                uri = "/google-cloud-logging/test",
+                method = 'POST',
+                upstream = {
+                    type = "roundrobin",
+                    nodes = {
+                        ["127.0.0.1:12001"] = 1
+                    }
+                },
+                plugins = {
+                    ["google-cloud-logging"] = {
+                        auth_file = "t/plugin/google-cloud-logging/config.json",
+                        inactive_timeout = 1,
+                        batch_max_size = 1,
+                        include_req_body = true,
+                    }
+                }
+            }
+            local t = require("lib.test_admin").test
+            local code, body = t('/apisix/admin/routes/1', ngx.HTTP_PUT, config)
+
+            if code >= 300 then
+                ngx.status = code
+                ngx.say(body)
+                return
+            end
+
+            ngx.say(body)
+        }
+    }
+--- response_body
+passed
+
+
+
+=== TEST 27: check request body included in log entry
+--- request
+POST /google-cloud-logging/test
+{"bodyItem": "something"}
+--- error_log eval
+qr/gcp logs body entries/
```

Comment:
Could we also check if the logged body is exactly the same as `{"bodyItem": "something"}`?